app/forms/form_navigation.rb

```diff
@@ -33,7 +33,7 @@ def initialize(controller)
   def next
     return unless index
 
-    controller_until_end = controllers[index + 1..-1]
+    controller_until_end = controllers[index + 1..]
```

Comment:
What's the effect of making this range to be endless?

Reply:
Answer: this is a (new?) style suggestion from Rubocop.

Reply:
IIRC, these two lines have the exact same effect....I don't think this changes anything programmatically...purely code style.